requirements.txt

```diff
@@ -1,9 +1,9 @@
-pybtex==0.20.1
-Pygments>=2.0,<3
-Sphinx==1.3.6
-sphinx_rtd_theme==0.1.9
-sphinxcontrib-bibtex==0.3.3
-numpy==1.11.0
-matplotlib==1.5.1
+pybtex
```

Comment:
Yes, several of those, do not remember exactly which.  From the history of this file, I guess only `Pygments<3` would imply "no Python 3". I wonder if you can just say `Python<3`, directly.